package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Intercept imports in Node.js",
   "main": "index.js",
   "scripts": {
-    "test": "c8 --check-coverage --lines 85 imhotap --runner 'node test/runtest' --files test/{hook,low-level,other,get-esm-exports}/*",
+    "test": "c8 --check-coverage --lines 80 imhotap --runner 'node test/runtest' --files test/{hook,low-level,other,get-esm-exports}/*",
```

Comment:
Why did the coverage drop?

Reply:
Coverage is going to be low on versions that aren't running some of the new code and thus tests will always fail